controllers/installation_controller.go

```diff
@@ -165,10 +173,15 @@ func (r *InstallationReconciler) Reconcile(ctx context.Context, req ctrl.Request
 	}
 
 	log.V(Log4Debug).Info("Reconciliation complete: A porter agent has been dispatched to apply changes to the installation.")
-	if r.PorterGRPCClient != nil {
-		return r.CheckOrCreateInstallationOutputsCR(ctx, log, inst)
+	var conn *grpc.ClientConn
```

Comment:
This is a dupe of 127:135, can we just have CheckOrCreateInstallationOutputsCR handle the connection establishment as well?